praw/exceptions.py

```diff
@@ -36,6 +36,18 @@ class ClientException(PRAWException):
     """Indicate exceptions that don't involve interaction with Reddit's API."""
 
 
+class DuplicateReplaceException(ClientException):
+    """Indicate exceptions that involve the replacement of MoreComments."""
+
+    def __init__(self):
+        """Instantize the class."""
+        super().__init__(
+            "A duplicate comment has been detected. Are you attempting to call"
+            " ``replace_more_comments`` more than once or after a comment "
```

Comment:
Just to be clear, calling `replace_more_comments` after a refresh by itself shouldn't be an issue. It's only an issue if a comment has already been replaced, right? If that's correct, then I think the latter part of the comment `or after a comment refresh` shouldn't be part of the message.

Reply:
But the original exception was caused by a refresh and then a replace, right?

Reply:
The `refresh` is only necessary in order to get the `More` comment object to show up again which can then be replaced. Refreshing isn't the root problem.

Reply:
Then what was the cause of the issue the PR tries to fix?

Reply:
I assumed it was the refresh but I might be wrong

Reply:
The refresh is necessary to get the `More` instance again. The root cause is replacing the same More instance more than once.

Reply:
Ahh. Maybe I should make a small disclaimer about that, like that using comment.refresh to obtain another More instance.

Reply:
Here's simpler strs:

```
submission = next(reddit.front.top())
submission.comment_limit = 1
submission.comments[1].comments()
submission.comments.replace_more(limit=1)
```

Results in `AssertionError`

Reply:
Ahhh. Noted.